File: Iniciante/1011 - Esfera/JavaScript/1011.js
function main() {
  const input = require("fs").readFileSync("/dev/stdin", "utf8");
  const radius = parseFloat(input);
  const PI = 3.14159;
  const FRACTIONAL_CONSTANT = 4.0 / 3.0;
  const volumeSphere = FRACTIONAL_CONSTANT * PI * Math.pow(radius, 3);
  console.log(`VOLUME = ${volumeSphere.toFixed(3)}`);
}

main();

File: Iniciante/1011 - Esfera/Python/1011.py
def main():
    radius = float(input())
    PI = 3.14159
    FRACTIONAL_CONSTANT = 4.0 / 3.0
    volume_sphere = FRACTIONAL_CONSTANT * PI * (radius**3)
    print(f"VOLUME = {volume_sphere:.3f}")


if __name__ == "__main__":
    main()
